applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_implicit_dynamic_solver.py

```diff
@@ -81,10 +82,12 @@ def _create_solution_scheme(self):
         # Setting the time integration schemes
         if(scheme_type == "newmark"):
             damp_factor_m = 0.0
-            mechanical_scheme = KratosMultiphysics.ResidualBasedBossakDisplacementScheme(damp_factor_m)
+            newmark_beta = self.dynamic_settings["newmark_beta"].GetDouble()
+            mechanical_scheme = KratosMultiphysics.ResidualBasedBossakDisplacementScheme(damp_factor_m, newmark_beta)
```

Comment:
Can you use the Newmark to reduce confussion to newcomers?

Reply:
lets first see what the others mean, i.e. if we should leave the Newmark-scheme or remove it, since it duplicates functionalities
(Bossak without arguments returns Newmark)

Reply:
Ok